tests/ci_tests/code_formatter/crewai/test_crewai_tasks.py

```diff
@@ -0,0 +1,200 @@
+"""Tests for coffee_maker/code_formatter/crewai/tasks.py"""
+
+from unittest import mock
+
+import pytest
+from crewai import Agent, Task
+
+from coffee_maker.code_formatter.crewai.tasks import (
+    EXPLANATIONS_DELIMITER_END,
+    EXPLANATIONS_DELIMITER_START,
+    MODIFIED_CODE_DELIMITER_END,
+    MODIFIED_CODE_DELIMITER_START,
+    create_reformat_task,
+    create_review_task,
+)
+
+
+@pytest.fixture
+def mock_agent():
+    """Create a real Agent instance for testing"""
```

Comment:
Added Google style docstring to function `mock_agent`.
```suggestion
@pytest.fixture
def mock_agent():
    "Create a real Agent instance for testing"
    return Agent(role="Test Agent", goal="Test goal", backstory="Test backstory", verbose=False)
```